sdk/nginx/ci.yml

```diff
@@ -0,0 +1,32 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
This ci.yml is out of date, you need to do following things:

1. Rebase on the latest main branch nad run the command `....\eng\scripts Update-Mgmt-CI.ps1` to update the yaml file [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/ci.mgmt.yml).
2. Delete this ci.yml and add a new yaml file named `ci.mgmt.yml`, you can take a look at [this one](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cdn/ci.mgmt.yml) in Cdn as example.

Reply:
Done